plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StandardFacetInstallDelegate.java

```diff
@@ -75,6 +75,7 @@ private AppEngineRuntimeInstallJob(IFacetedProject facetedProject) {
       super(Messages.getString("appengine.install.runtime.to.project", // $NON-NLS$
           facetedProject.getProject().getName()));
       this.facetedProject = facetedProject;
+      setRule(facetedProject.getProject());
```

Comment:
Hmm... interesting. I *think* we have a deadlock situation like below (I think the job "Configuring for JavaScript" is the first `ConvertJob`, because we resume the suspender at the end of "Install App Engine runtimes"), but the deadlock is resolved after some time automatically:

![selection_002](https://user-images.githubusercontent.com/10523105/27611050-b6a40c28-5b5e-11e7-8f83-e90e68e16a2d.png)

Not sure if this is because the Job manager has an ability to resolve deadlocks. But then, I don't see any log about it, so this may not be a deadlock situation at all.